articles/techniques/local-variables.md

```diff
@@ -69,7 +69,7 @@ For this reason, it is a good practice to make functions and operations which us
 
 ## Deconstruction ##
 
-In addition to assigning a single variable, the `let` and `mutable` keywords - or in fact any other binding construct - also allow for unpacking the contents of a [tuple type](#tuple-types).
+In addition to assigning a single variable, the `let` and `mutable` keywords - or in fact any other binding construct - also allow for unpacking the contents of a [tuple type](..\language\type-model#tuple-types).
```

Comment:
In general, I recommend xref links to articule UIDs rather than relative paths, as the former tend to be much more stable when reorganizing and refactoring content.
```suggestion
In addition to assigning a single variable, the `let` and `mutable` keywords - or in fact any other binding construct - also allow for unpacking the contents of a [tuple type](xref:microsoft.quantum.language.type-model#tuple-types).
```